assets/airports/eidw.json

```diff
@@ -875,136 +1072,578 @@
     ],
     "runways":[
         {
-            "name":        ["10", "28"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N53d25m20.75", "W6d17m24.27", "242ft"], ["N53d25m12.94", "W6d15m02.08", "202ft"]],
-            "length":      2.637,
-            "ils":         [true, true]
+            "name": ["10", "28"],
+            "end": [
+                ["N53d25m20.75", "W6d17m24.27", "242ft"],
+                ["N53d25m12.94", "W6d15m02.08", "202ft"]
+            ],
+            "ils": [true, true]
         },
         {
-            "name":        ["16", "34"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
-            "length":      2.072,
-            "ils":         [true, true]
+            "name": ["16", "34"],
+            "end": [
+                ["N53d26m13.16", "W6d15m43.12", "217ft"],
+                ["N53d25m11.66", "W6d14m58.54", "202ft"]
+            ],
+            "ils": [true, true]
         }
     ],
     "sids": {
-        "SUROX": {
-            "icao": "SUROX",
-            "name": "Surox",
-            "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+        "ABIDO": {
+            "icao": "ABIDO",
+            "name": "Abido",
+            "suffix": {"10":"2F", "16":"1P", "28":"1B", "34":"1H"},
             "rwy": {
-                "10": [["DW551", "A30+"], "LEMTA", "DODIG"],
-                "28": [["OE", "A30+"], "DWW04"],
-                "16": [["DWS77", "A30+"], "BATED", "ERUDA", "DODIG"],
-                "34": [["DWN26", "A30+"], "DWW06", "DWW04"]
+                "10": ["DW679", "IRDEX", "ERVAD"],
+                "16": ["DWS81", "NEPOD"],
+                "28": ["DWS80", "DW682", "NEPOD"],
+                "34": ["OE", "DWW02", "BAL", "NEPOD"]
             },
-            "body": ["SUROX"],
-            "draw": [["OE", "DWW04"], ["ERUDA", "DODIG", "SUROX"], ["DWW06", "DWW04", "SUROX"]]
+            "body": [],
+            "exitPoints": {
+                "PESIT": ["ABIDO", "PESIT"]
+            },
+            "draw": [
+                ["DW679", "IRDEX"],
+                ["DWS81", "NEPOD"],
+                ["ABIDO"]
+            ]
         },
-        "LIFFY": {
-            "icao": "LIFFY",
-            "name": "Liffy",
-            "suffix": {"10":"2E", "28":"5A", "16":"5M", "34":"5G"},
+        "BEPAN": {
```

Comment:
I somehow managed to convince myself that BAMLI had no SID's :thinking: Anyway it's fixed now 👍